### ISSUE TITLE
Counting and taking screenshot for  special results in FP Gacha

### ISSUE THREAD

**Issue by item4** [Fate-Grand-Automata/FGA#1339]
### Preparation

- [X] No one has asked for this feature before in other issues (even the closed ones)

### Describe the feature

I want to know how much Altria Pendragon (Lily) and Aŋra Mainiiu during my FP Gacha running.

Example output what I want.

```
Stop: Inventory Full
- Rolling 1000 round(s).
- Get R4 Altria Pendragon (Lily) x3
- Get R0 Aŋra Mainiiu x1
```

**Comment by reconman** [comment]
The chances are so slim that it's not worth implementing in my opinion.

**Comment by item4** [comment]
Yes. but I want to remain that special rare situation by log or screenshot.
